cmake/Reprojucer.cmake

```diff
@@ -2640,11 +2675,13 @@ endfunction()
 
 function(__set_JucePlugin_Build_defines target target_type)
 
-  # See XCodeProjectExporter::Target::getTargetSettings()
```

Comment:
Same question here about the comments

Reply:
Same answer. The code moved and the comment is not accurate anymore.